docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -196,6 +196,20 @@ See more details in the [IOperation test hook](https://github.com/dotnet/roslyn/
 `C:\Source\roslyn> cd ..`
 `C:\Source> dotnet-format analyzers .\roslyn\Compilers.sln --diagnostics=RS0016 --no-restore --include-generated -v diag`
 
+## Replicating Failures in the Used Assemblies leg
```

Comment:
Minor: Should this section use `###` instead?

Reply:
Sure. I'll make the change